tasks/task_two.js

```diff
@@ -0,0 +1,8 @@
+const num1 = 10
+const num2 = 5
+
+console.log(num1 + num2)
```

Comment:
Try assigning the result of your expression to a variable, so that you can always reference it.

Reply:
Changes made.